utilities/rpp-unittests/HIP_NEW/Tensor_hip_pkd3.cpp

```diff
@@ -176,6 +176,9 @@ int main(int argc, char **argv)
     case 31:
         strcpy(funcName, "color_cast");
         break;
+    case 34:
+        strcpy(funcName, "lut");
```

Comment:
Lets remove updates to these HIP_NEW and HOST_NEW folders since they will be deleted after all functions are integrated into test_suite folder

Reply:
Removed them